repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -135,6 +137,42 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
       }
     }
 
+    def getInventory(headerResult: ValkyrieResult): ValkyrieResult = {
+      def parseInventory(inputStream: InputStream): Try[UserPermissions] = {
+
+        @tailrec
+        def parseJson(deviceToPermissions: List[DeviceToPermission], values: List[JsValue]): UserPermissions = {
+          if (values.isEmpty) {
+            UserPermissions(List(ACCOUNT_ADMIN).toVector, deviceToPermissions.toVector)
```

Comment:
I made it an empty `Vector`, but for now I left it returning `UserPermissions` so that it remains compatible with a `ValkyrieResult`.